Zend/tests/018.phpt

```diff
@@ -18,7 +18,7 @@ var_dump(constant("TEST_CONST2"));
 echo "Done\n";
 ?>
 --EXPECT--
-Undefined constant ''
+Undefined constant 
```

Comment:
The new version is consistent with the error messages of undefined variable, but falls short in this case. Maybe we should retain the quotes for the empty string name?